api/datastore/sql/migrations/18_create_new_calls.go

```diff
@@ -0,0 +1,49 @@
+package migrations
+
+import (
+	"context"
+
+	"github.com/fnproject/fn/api/datastore/sql/migratex"
+	"github.com/jmoiron/sqlx"
+)
+
+func up18(ctx context.Context, tx *sqlx.Tx) error {
+	dropRoutesTable := `DROP TABLE routes;`
+	_, err := tx.ExecContext(ctx, dropRoutesTable)
+	if err != nil {
+		return err
+	}
+
+	dropCallsTable := `DROP TABLE calls;`
+	_, err = tx.ExecContext(ctx, dropCallsTable)
+	if err != nil {
+		return err
+	}
+
+	createQuery := `CREATE TABLE IF NOT EXISTS calls (
+	created_at varchar(256) NOT NULL,
+	started_at varchar(256) NOT NULL,
+	completed_at varchar(256) NOT NULL,
+	status varchar(256) NOT NULL,
+	id varchar(256) NOT NULL,
+	app_id varchar(256) NOT NULL,
+	fn_id varchar(256) NOT NULL,
+	stats text,
+	error text,
+	PRIMARY KEY (id)
+);`
+	_, err = tx.ExecContext(ctx, createQuery)
+	return err
+}
+
+func down18(ctx context.Context, tx *sqlx.Tx) error {
+	return nil
```

Comment:
down must revert a change, so it has to put back a table, ideally with data (not sure how this can be done, but still matters)